ui/packages/consul-ui/tests/acceptance/dc/services/index.feature

```diff
@@ -96,11 +101,12 @@ Feature: dc / services / index: List Services
     ---
     Then the url should be /dc-1/services
     And the title should be "Services - Consul"
-    Then I see 2 service models
+    Then I see 3 service models
```

Comment:
I'm seeing this test fail locally still showing 2 service models. I can try and look into it to help out later if I have some time after some meetings